index.bs

```diff
@@ -2127,6 +2127,20 @@ it also ensures a consistent development experience for the users.
 A good example where debuggability was defined as
 part of the specification is [Web Animations](https://www.w3.org/TR/web-animations-1/#use-cases).
 
+<h3 id="crypto">Use the best crypto, and expect it to evolve</h3>
+
+Use only cryptographic algorithms
+that have been impartially reviewed by security experts,
+and make sure your choice of algorithm is proven, and up-to-date.
+
+Not only do they become obsolete or insecure,
+cryptographic protocols and algorithms also evolve quickly.
+Therefore, features should be designed
+so that they are usable in an algorithm-independent manner.
+For example, [WebCrypto](https://www.w3.org/TR/WebCryptoAPI/#scope-algorithms)
+is designed so that the underlying algorithm
```

Comment:
```suggestion
<h3 id="crypto">Use the best crypto, and expect it to evolve</h3>

Use only cryptographic algorithms
that have been impartially reviewed by security experts,
and make sure your choice of algorithm is proven, and up-to-date.
Not only do they become obsolete or insecure,
cryptographic protocols and algorithms also evolve quickly.
```

Suggestion to remove the part that @sleevi raised concerns with (https://github.com/w3ctag/design-principles/pull/310/files#r630611213) but keep the main essence of the principle.